datasets/beheerkaart_cbs_grid/beheerkaart_cbs_grid.json

```diff
@@ -0,0 +1,119 @@
+{
+  "type": "dataset",
+  "id": "beheerkaart_cbs_grid",
```

Comment:
De id zou denk ik camelCase moeten. Maar, ik weet niet of we dat al in een dataset hebben op dit moment. De vraag is dus of het goedgaat als we het veranderen.

Reply:
Ja idd, ik heb hiervoor afgekeken hoe de eerdere beheerkaart_basis is aangemaakt, en daar was de tabel niet in camelCase. Maar volgens mij heb je wel gelijk. Ik zet deze in camelCase en kijk of dit goed gaat. Zou natuurlijk wel moeten werken.